standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/datasource/HikariCPDataSourceProvider.java

```diff
@@ -45,15 +45,19 @@ public class HikariCPDataSourceProvider implements DataSourceProvider {
 
   @Override
   public DataSource create(Configuration hdpConfig) throws SQLException {
+    int maxPoolSize = MetastoreConf.getIntVar(hdpConfig,
```

Comment:
Could we extract this into the default method in the interface since we always use the same conf - MetastoreConf.ConfVars.CONNECTION_POOLING_MAX_CONNECTIONS?